damus/Views/PostView.swift

```diff
@@ -65,7 +66,16 @@ struct PostView: View {
 
                 if !is_post_empty {
                     Button(NSLocalizedString("Post", comment: "Button to post a note.")) {
-                        self.send_post()
+                        if #available(iOS 16.0, *) {
```

Comment:
this logic shouldn't be duplicated in two places

Reply:
Refactored.